dialogue-clients/src/test/java/com/palantir/dialogue/otherpackage/DialogueClientsTest.java

```diff
@@ -169,9 +165,9 @@ void perHost_attributes_concurrencylimiter_metrics_to_the_right_host() {
                         .putServices(
                                 "my-service",
                                 PartialServiceConfiguration.builder()
-                                        .addUris("https://my-service-0.fake.palantir.com")
-                                        .addUris("https://my-service-1.fake.palantir.com")
-                                        .addUris("https://my-service-2.fake.palantir.com")
+                                        .addUris("https://127.0.0.1/0")
+                                        .addUris("https://127.0.0.1/1")
+                                        .addUris("https://127.0.0.1/2")
```

Comment:
Updated because the manifestation is slightly different -- this way we allow the actual client to progress as expected.
Without using localhost here, the dns resolution component reduces the target to no TargetUris as dns resolution fails.